src/test/java/temporary/tests/TestOpenPageChrome.java

```diff
@@ -25,7 +25,8 @@ public void openPage() {
         String url = getProperty(BASE_URL);
         driver.navigate().to(url);
 
-        var welcomeMsg = By.xpath("//div[@class='welcome_msg']");
+        var welcomeMsg = By.xpath("//div[@clas" +
```

Comment:
Accidental change probably, to be reverted:) You can do it by using git checkout <filePath>

Reply:
resolved